UI/data/locale.ini

```diff
@@ -65,10 +68,10 @@ Name=Svenska
 Name=Türkçe
 
 [zh-CN]
-Name=简体中文
+Name=中文简体
 
 [zh-TW]
-Name=繁體中文
```

Comment:
Based on feedback for the above proposed change for zh-CN, I'd guess this is also not an ideal change.